prow/manifests/overlays/metal3/config.yaml

```diff
@@ -2921,39 +2921,6 @@ presubmits:
                 value: "TRUE"
             image: docker.io/pipelinecomponents/markdownlint:0.13.0@sha256:9c0cdfb64fd3f1d3bdc5181629b39c2e43b6a52fc9fdc146611e1860845bbae0
             imagePullPolicy: Always
-    # name: {job_prefix}-{image_os}-e2e-integration-test-{capm3_target_branch}
-    - name: metal3-centos-e2e-integration-test-main
-      agent: jenkins
-      always_run: false
-      optional: true
```

Comment:
These are the actual prow job definitions. Without them you won't be able to trigger anything through prow.
The required status checks above are just things tide should make sure are there before merging. I.e. prow knows nothing about them except that they must be there

Reply:
Oh yes, of course. However they cannot be optional, and need to be branch by branch defined then, as they cannot be listed like this and be required. Prow won't allow that.